src/mongoose_user_stats.erl

```diff
@@ -2,11 +2,36 @@
 
 -author('aleksander.lisiecki@erlang-solutions.com').
 
--export([report/0]).
+-behaviour(mongoose_service).
+
+-export([start/1, stop/0]).
+
+-define(TAB_NAME, persistent_user_info).
```

Comment:
I think it would be perfectly fine to just name the table after the module, because the current name is a bit cryptic about the table purpose.